qdev_wrappers/customised_instruments/AlazarTech_ATS9360_ext.py

```diff
@@ -8,24 +8,30 @@ def __init__(self, name):
         self.add_parameter(name='seq_mode',
                            get_cmd=self._get_seq_mod,
                            set_cmd=self._set_seq_mode,
-                           vals=vals.Enum('on', 'off'))
+                           vals=vals.Enum('on', 'off'),
+                           get_cmd=get_parser_on_off,
+                           set_cmd=set_parser_on_off)
```

Comment:
I can't seem to spot that this is imported anywhere?

Reply:
you're right, let me fix that